packages/aws-cdk/lib/init-templates/app/dotnet/app.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# This script is configured in cdk.json to be used to
+# execute the CDK .NET app by the command-line toolkit.
+# The first argument will be used as argv[0]
+exec dotnet run src/HelloCdk/bin/Release/HelloCdk $@
```

Comment:
I don't think you need this script, I think you can just put `dotnet run src/HelloCdk/bin/Release/HelloCdk` in the `cdk.json` file

Reply:
✔️